boa_engine/src/syntax/ast/node/mod.rs

```diff
@@ -22,6 +22,7 @@ pub mod switch;
 pub mod template;
 pub mod throw;
 pub mod try_node;
+pub mod visit;
```

Comment:
Could we add a feature flag for this? I don't think we need it in normal operation, right? and we would avoid compiling it for most of the cases. We would need to add the feature to CI, though, to make sure we still test it.